packages/ember/tests/reexports_test.js

```diff
@@ -35,319 +35,315 @@ moduleFor(
   }
 );
 
-let allExports = [
-  // @ember/application
-  ['Application', '@ember/application', 'default'],
-  ['getOwner', '@ember/application', 'getOwner'],
-  ['onLoad', '@ember/application', 'onLoad'],
-  ['runLoadHooks', '@ember/application', 'runLoadHooks'],
-  ['setOwner', '@ember/application', 'setOwner'],
-
-  // @ember/application/instance
-  ['ApplicationInstance', '@ember/application/instance', 'default'],
-
-  // @ember/application/namespace
-  ['Namespace', '@ember/application/namespace', 'default'],
-
-  // @ember/array
-  ['Array', '@ember/array', 'default'],
-  ['A', '@ember/array', 'A'],
-  ['NativeArray', '@ember/array', 'NativeArray'],
-  ['isArray', '@ember/array', 'isArray'],
-  ['makeArray', '@ember/array', 'makeArray'],
-
-  // @ember/array/mutable
-  ['MutableArray', '@ember/array/mutable', 'default'],
-
-  // @ember/array/proxy
-  ['ArrayProxy', '@ember/array/proxy', 'default'],
-
-  // @ember/canary-features
-  ['FEATURES.isEnabled', '@ember/canary-features', 'isEnabled'],
-
-  // @ember/component
-  ['Component', '@ember/component', 'default'],
-  ['_componentManagerCapabilities', '@ember/component', 'capabilities'],
-  ['_getComponentTemplate', '@ember/component', 'getComponentTemplate'],
-  ['_setComponentManager', '@ember/component', 'setComponentManager'],
-  ['_setComponentTemplate', '@ember/component', 'setComponentTemplate'],
-
-  // @ember/component/helper
-  ['Helper', '@ember/component/helper', 'default'],
-  ['Helper.helper', '@ember/component/helper', 'helper'],
-
-  // @ember/component/template-only
-  ['_templateOnlyComponent', '@ember/component/template-only', 'default'],
+import * as test0 from '@ember/application';
```

Comment:
This whole file deserves to be deleted as soon as possible, I don't think it's worth spending any time making it prettier. I did this change in a semi-automated way because it's... a lot.